### PR TITLE
[script.pystone.benchmark] 1.1.0

### PR DESCRIPTION
**Pull request by Lunatixz**
Test your devices python performance.

### Checklist:

- [x] My code follows the [add-on rules](http://kodi.wiki/view/Add-on_rules) and [piracy stance](http://kodi.wiki/view/Official:Forum_rules#Piracy_Policy) of this project. 
- [x] I have read the [CONTRIBUTING](https://github.com/xbmc/repo-scripts/blob/master/CONTRIBUTING.md) document
- [x] Each add-on submission should be a single commit with using the following style: [script.foo.bar] v1.0.0

Additional information :
- Submitting your add-on to this specific branch makes it available to any Kodi version equal or higher than the branch name with the applicable Kodi dependencies limits.
- [add-on development](http://kodi.wiki/view/Add-on_development) wiki page.
- Kodi [pydocs](http://kodi.wiki/view/PyDocs) provide information about the Python API
- [PEP8](https://www.python.org/dev/peps/pep-0008/) codingstyle which is considered best practise but not mandatory.
- This add-on repository has automated code guideline check which could help you improve your coding. You can find the results of these check at [Codacy](https://www.codacy.com/app/Kodi/repo-scripts/dashboard). You can create your own account as well to continuously monitor your python coding before submitting to repo.
- Development questions can be asked in the [add-on development](http://forum.kodi.tv/forumdisplay.php?fid=26) section on the Kodi forum.

### CODE REVIEW

**Comment by Razzeee** [comment]
Just a little comment, nothing that keeps us from merging. 

It would probably improve things if you move the content of default.py into another file and keep default.py to a minimum. 
Otherwise your addon  will take more time to startup due to compilation.

**Comment by Lunatixz** [comment]
@Razzeee okay, THX

File: script.pystone.benchmark/addon.xml [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+        <provides>executable</provides>
+    </extension>
+    <extension point="xbmc.addon.metadata">
+        <summary lang="en_GB">Test your devices python performance.</summary>
+        <description lang="en_GB">How fast does your device run python code? Pystone evaluates CPU performance running in a python environment.</description>
+        <license>The MIT License (MIT)</license>
+        <platform>all</platform>
+        <source>https://github.com/Lunatixz/XBMC_Addons/tree/master/script.pystone.benchmark</source>
+        <email>pseudotvlive@gmail.com</email>
+        <forum>http://forum.kodi.tv/</forum>
```
Comment (enen92) [inline script.pystone.benchmark/addon.xml]: Please use a specific thread

**Comment by enen92** [comment]
I don't think we allow Kodi logos in third party plugins (fanart and icon) - for them not to be confused as developed by team kodi. Can you change them for something else please?

**Comment by Lunatixz** [comment]
THX, Is this a new rule? there are currently a number of plugins in the official repo using kodis logo as the icon.... also can you please expand on the use of the term "Third-Party" I thought the whole point of submitting to the official repo was to become "First-Party".

File: script.pystone.benchmark/addon.xml (hunk 1 above)
Comment (Lunatixz) [inline script.pystone.benchmark/addon.xml]: I'll remove the reference completely, I don't think there is a need to open a specific thread for this script.... so I left the general forum as a reference. :)

**Comment by Razzeee** [comment]
There are some addons done by "Team Kodi" those can use the logo.

**Comment by enen92** [comment]
All plugins are third party even the ones on the official repository. However, there are some that are developed by Team Kodi itself. Allowing other plugins to use Kodi's branding is something we want to avoid just as a means to avoid confusion. It is better for us and also for the ones who develop plugins. Hope that's something you can do with minimal effort @Lunatixz

**Comment by Lunatixz** [comment]
@Razzeee I don't believe that's the case for all of them... seeing how this is a newer rule? Some older plugins and including a majority of skins.... may have been grandfathered in? 
https://kodi.tv/addon/program-add-ons-scripts/tvshow-time

@enen92 effort isn't a concern... gaining knowledge around Kodi and its rules are. I don't want you to think I question rules as a act of defiance., I simply want to understand the "Why" to reduce issues down the road. I understand the "Why" regarding the use of the Kodi logo... Just a shame thought I came up with a snazzy logo ;)

**Comment by Razzeee** [comment]
Yeah, I would have to look if there really is a rule. We might just have gotten more sensible to other parties using our logo and being thought of as beeing us.

**Comment by Razzeee** [comment]
@Lunatixz 
Seems like it's not allowed according to the rules, hope I'm reading this right
`Don't mix logos with Kodi's logo; it's obvious that we're talking about Kodi here.`

Btw transperency is also not allowed in the icon.png 
See: http://kodi.wiki/view/Add-on_structure#icon.png

And I don't know if you have asked the person who created the fanart if you can include it? Probably not a problem, but we would like to be on the safe side of things.


Anyway addon sounds cool, hope you don't get demotivated by this minor setback ;)
Might get really interesting if we ever get python3 and are able to compare the performance of the versions :)

**Comment by Lunatixz** [comment]
@Razzeee okay, i'll replace the icon and fanart ASAP. THX for the helpful info.

**Comment by enen92** [comment]
@Lunatixz a fanart is mandatory according to the submission rules. Can you please add one? Sorry for the, sometimes, boring process

**Comment by enen92** [comment]
Thanks for adjusting @Lunatixz

**Pull request merged**